sections/introduction.tex

```diff
@@ -0,0 +1,61 @@
+\section{Introduction} 
+%% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.
+
+
+Many news articles are produced every day and the need for searching the news is becoming a more prominent and difficult task.
+A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus. 
+While some engines use simple algorithms, such as finding all articles that include a given search term and sorting by date, this may not always give useful results.
+Information retrieval within search engines apply many different techniques for either ranking or filtering documents\cite{google_pagerank2006}.
+
+In order to rank or filter these documents, some information is needed to support the underlying ranking or filtering.
+A popular way to extract this information is through the use of topic modeling.
```

Comment:
is it the information usefull for ranking or filtering or is it just usefull information?

Reply:
citation needed